src/app/public/modules/select-field/fixtures/select-field.component.fixture.html

```diff
@@ -12,5 +12,7 @@
     [singleSelectPlaceholderText]="singleSelectPlaceholderText"
     [pickerHeading]="pickerHeading"
     [ngModel]="formData.modelValue"
-    (ngModelChange)="onModelChange($event)">
+    (blur)="onTouch()"
```

Comment:
Totally a minor thing but any way we could rename this fixture method to `onBlur`? It makes sense as it is now to me today but I think it may throw me off a few months from now when I come back to this code.

Reply:
sure